google-cloud-storage/lib/google/cloud/storage/file/signer_v4.rb

```diff
@@ -164,6 +164,7 @@ def escape_special_unicode str
 
           def required_fields issuer, time
             {
+              "bucket" => @bucket_name,
```

Comment:
@quartzmo does this output a field `bucket` in returned fields? We may have already discussed this but double checking.

Reply:
Gotcha, yes please, could you update it to only include the name in the policy only and not return it as a field.?

Reply:
Yes, will do.

Reply:
Fixed in #5803.